src/bin/options.ts

```diff
@@ -53,6 +53,21 @@ const clientPort = new commander.Option(
   .env('PK_CLIENT_PORT')
   .default(0);
 
+const ingressHost = new commander.Option(
+  '-ih, --ingress-host <address...>',
+  'ReverseProxy ingress host',
+)
+  .env('PK_INGRESS_HOST')
+  .default('0.0.0.0');
```

Comment:
This was previously using the default value as specified in `src/config.ts`:

```ts
const config = {
  ... ,
  defaults: {
    ingressHost: '0.0.0.0',
    ingressPort: 0,
    clientHost: 'localhost',
    clientPort: 0,
  },
```

@tegefaulkes mentioned that this change came from `master`. Are we still using the `config.ts` file for keeping track of all of our static default values, and just importing that file as required?

Reply:
@CMCDragonkai

Reply:
Right now all defaults were set in the PolykeyAgent. I think this is fine. Config is more for global static configuration. It would be weird to see the default IP to be in the config. It would look static.

So right now we have 3 places to have defaults. PolykeyAgent, PolykeyClient, and bin/options.ts. The config should be used for static unchanging things. Exception is your seed nodes.